VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -101,65 +101,62 @@
 
             <LinearLayout
                 android:id="@+id/last_access_bar"
```

Comment:
There are too many `LinearLayout` nesting in this layout, but I guess we can leave it for now.